### ISSUE TITLE
Duplicate definitions from HTML spec

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/webref#780]
Something odd has happened in the last few days and duplicate definitions have started appearing in the data for HTML, for example:

"sandboxed navigation browsing context flag"
https://respec.org/xref/?term=sandboxed+navigation+browsing+context+flag

(see other "context flags" too)

One points to:
 * [origin.html#sandboxed-navigation-browsing-context-flag](https://html.spec.whatwg.org/multipage/browsers.html#sandboxed-navigation-browsing-context-flag)

The other points to:
 * [browsers.html#sandboxed-navigation-browsing-context-flag](https://html.spec.whatwg.org/multipage/browsers.html#sandboxed-navigation-browsing-context-flag)

(the second one is correct)

This is confusing ReSpec, as ReSpec assumes there is only one definition per document. 
 
I checked in HTML. and it is indeed only defined once.

**Comment by tidoust** [comment]
Ah yes, the system is not quite good at propagating changes in the list of pages of a multi-pages spec.

This was triggered by the recent [navigation and session history rewrite](https://github.com/whatwg/html/commit/0a97a81da77bc4cb0ab5b16420605ca001ff5b17) in HTML, which modifies the list of pages (in particular, `origin.html` was dropped, and now redirects to `browsers.html`).

Changes in the list of pages of a multi-pages spec take some time to propagate because the list of pages is captured in browser-specs, not during the crawl itself. I published new browser-specs packages with the updated list of pages, released a new version of Reffy accordingly, and triggered a new crawl. Duplicate definitions are now gone.

We should probably drop such duplicates during the crawl so that you don't have to in ReSpec. I created #783 to track this.
